packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -51,6 +52,7 @@ export const getComponentCss = (
         ...colorSchemeStyles,
         ...hostHiddenStyles,
       }),
+      ':host(:only-of-type)': { borderBottom: addImportantToRule(0) },
```

Comment:
Can't we nest this within `:host` above to reduce redundancy in selector and extra `addImportantToRule` call?